src/app/pages/apps/chart/chart-account.component.ts

```diff
@@ -262,16 +271,30 @@ export class ChartAccountComponent implements OnInit, OnDestroy {
             }
             if (data.errorDescription) {
               this._snackBar.open(data.errorDescription, 'Close', {
-                duration: 5000,
+                duration: 3000,
               });
             } else {
-              this._snackBar.open('Chart of Accounts added successfully.', 'Close', {
-                duration: 5000,
+              let snackBarRef = this._snackBar.open('Chart of Accounts added successfully.', 'Close', {
+                duration: 3000,
               });
-              this.form.reset();
-              this.salesData.sourceCollection = [];
-            }
+              snackBarRef.afterDismissed().subscribe(() => {
+                window.location.reload();
+            });
+          }
           });
         }
   }
-}
+
+  private areFormsEqual(formValue1: any, formValue2: any): boolean {
```

Comment:
![image](https://github.com/vernienamca/FAIS.Web/assets/151013941/21000b42-b69a-451d-b119-8e2d16e9c42a)

reference for this method.. Converted it to string to compare